docs/reference/cli.rst

```diff
@@ -173,3 +173,15 @@ Example::
 
     This command publishes drafts. You should review drafts before using this
     command, because they will become public.
+
+**********************
+Maintenance and repair
+**********************
+
+.. _fix-mptt:
+
+``fix-mptt``
```

Comment:
I can't see a way around it - there *weren't* any docs previously, which is why I added this.